### PR TITLE
Fix Log Entry to Config File Bug

### PR DESCRIPTION
**Pull request by Lilferrit**
Fixed a bug that would write the log entries to the config file instead of the log file when running the `configure` command.

### CODE REVIEW

File: casanovo/casanovo.py [hunk 1]
```diff
@@ -337,21 +336,46 @@ def version() -> None:
 
 
 @main.command()
+@click.option(
```
Comment (bittremieux) [inline casanovo/casanovo.py:339]: **question:** Could we maybe make two sets of shared commands? One that has these three shared options related to output files, and the other with the remaining options that are in the current `_SharedParams`. Then we don't need to duplicate the first ones and combine those two sets of shared commands where necessary. That would avoid having to redefine those output options here.

File: casanovo/casanovo.py [hunk 2]
```diff
@@ -337,21 +336,46 @@ def version() -> None:
...
+        output_dir, output_root, force_overwrite, "info"
+    )
+    config_fname = output_root if output_root is not None else "casanovo"
+    config_fname = Path(config_fname).with_suffix(".yaml")
+    if not force_overwrite:
+        utils.check_dir_file_exists(output_path, str(config_fname))
+
+    config_path = str(output_path / config_fname)
+    Config.copy_default(config_path)
+    logger.info(f"Wrote {config_path}\n")
```
Comment (bittremieux) [inline casanovo/casanovo.py:378]: **nitpick:** Is the newline at the end necessary?

**Review by bittremieux — COMMENTED** [review]
Looks good, just some considerations.

As you mentioned on Slack, I think it would also be good to add an integration test that checks whether the produced config file is actually compatible (i.e. you can run Casanovo with it).

File: casanovo/casanovo.py (hunk 1 above)
Comment (Lilferrit) [reply casanovo/casanovo.py]: Sounds good, I split the file IO commands into a different class to avoid redefining them.

**Comment by Lilferrit** [comment]
Sounds good, I added additional test cases for the `configure` sub-command as well splitting the file IO options into a different class.

**Review by bittremieux — APPROVED** [review]

**Pull request merged**